components/brave_ads/browser/ads_service_impl.h

```diff
@@ -239,7 +243,7 @@ class AdsServiceImpl : public AdsService,
   bat_ads::mojom::BatAdsServicePtr bat_ads_service_;
 
   NotificationInfoMap notification_ids_;
-  std::map<const net::URLFetcher*, ads::URLRequestCallback> fetchers_;
+  std::unordered_set<network::SimpleURLLoader*> url_loaders_;
```

Comment:
nit: `Generally avoid std::unordered_set and std::unordered_map...` from `base/containers/README.md`

Reply:
Done.